.pipelines/pipeline.yaml

```diff
@@ -322,24 +322,39 @@ stages:
       displayName: Cilium
       pipelineBuildImage: "$(BUILD_IMAGE)"
       testDropgz: ""
+      clusterType: swift-no-kube-proxy-up
       clusterName: "ciliume2e"
+      vmSize: Standard_B2ms
+      k8sVersion: ""
+      windowsOsSku: ""
+      dependsOn: 'containerize'
 
   - template: singletenancy/cilium-overlay/cilium-overlay-e2e-job-template.yaml
     parameters:
       name: "cilium_overlay_cilium_e2e"
       displayName: Cilium on AKS Overlay
       pipelineBuildImage: "$(BUILD_IMAGE)"
       testDropgz: ""
+      clusterType: overlay-no-kube-proxy-up
       clusterName: "overlaye2e"
+      vmSize: Standard_B2ms
+      k8sVersion: ""
+      windowsOsSku: ""
+      dependsOn: 'containerize'
```

Comment:
Show we depend on `validate1` so that we know the change is upto date with master

Reply:
As long as the respective stages are dependent on the same stages that validate1 and validate2 are then we know those respective stages are up to date with master. The cluster creation stage matches with validate1 and E2E matches with validate2 with the additional dependency that cluster creation for the respective E2E stage is complete.